src/main/java/com/gluonhq/substrate/target/IosTargetConfiguration.java

```diff
@@ -90,10 +139,30 @@ public String getAdditionalSourceFileLocation() {
         return iosAdditionalSourceFiles;
     }
 
+    @Override
     List<String> getTargetSpecificObjectFiles() throws IOException {
         Path gvmPath = paths.getGvmPath();
         Path objectFile = FileOps.findFile(gvmPath, "llvm.o");
-        return Arrays.asList(objectFile.toAbsolutePath().toString());
+        return Collections.singletonList(objectFile.toAbsolutePath().toString());
+    }
+
+    @Override
+    List<String> getJavaFXReflectionClassList() {
+        List<String> answer = super.getJavaFXReflectionClassList();
```

Comment:
Depending on how many times this is executed - it can produce different results I think. Cleaner way might be to add both lists to a new list and return that. Also we seem to do this kind of thing many times, should we think of caching for performance reasons?

Reply:
I'm not following?
`super` returns already a static list `javafxReflectionClassList`, and we combine it with the platform list (`javafxReflectionIosClassList` in this case) into the resulting list.

Reply:
I thought the original list is updated and then has all the new elements. if this method is called more then once - it will be updated multiple times.

Reply:
Yes, you are right, the original list is updated and a second call will increase it with duplicated items. I'll fix this.

Reply:
It should be fixed now

Reply:
I think I saw many places like that before :)